lib/containers/container_images.pm

```diff
@@ -175,6 +175,32 @@ sub test_opensuse_based_image {
     }
 }
 
+sub verify_userid_on_container {
```

Comment:
this test does not make sense for docker currently , right ?

Reply:
i havent tested it but i cant see the reason why it wouldnt